tutorials/clustering.md

```diff
@@ -0,0 +1,76 @@
+# Tutorial: Scaling out TimescaleDB
+
+>:WARNING: Running TimescaleDB in a multi-node setup is currently in PRIVATE BETA.
+This method of deployment is not meant for production use.
+
+TimescaleDB can be run in a multi-node setup, with one primary access node distributing
+writes and queries to multiple data nodes. For more information on the actual
+underlying architecture, you can view our [architecture][architecture] section.
+
+In this tutorial, we will show how you can create a distributed hypertable within
+a distributed database. For simplicity, we will set up one primary access node
+and two data nodes all on local host. However, in the case of scaling out, you would
+want the access node and data nodes to be distinct TimescaleDB instances.
+
+Prerequisites:
+- 3 instances of TimescaleDB installed (private BETA branch only)
+- A superuser set up on all 3 instances of TimescaleDB
+- Network connectivity between the 3 TimescaleDB instances
+
+## Define the topology for your distributed database
+
+Now that you have 3 running instances of TimescaleDB, you can go ahead and set up your
+distributed database. Connect to the instance that you would like to act as the access node,
+and create a new database there.
+
+```sql
+CREATE DATABASE multinode;
+\c multinode
+CREATE EXTENSION timescaledb;
+```
+
+Now, you can tell your access node that currently contains the multinode database to add
+data nodes that it can use to store distributed hypertables.
+
+```sql
+SELECT add_data_node('dn1',host=>'<address>', if_not_exists=>'true',
+bootstrap_user=>'<postgres_user>', bootstrap_password=>'<postgres_password>');
+SELECT add_data_node('dn2',host=>'<address>', if_not_exists=>'true',
+bootstrap_user=>'<postgres_user>', bootstrap_password=>'<postgres_password>');
+```
+
+You can check whether or not the data nodes have been connected successfully by
+using our informational views.
+
+```sql
+SELECT * FROM timescaledb_information.data_node;
+```
+
+Now that we've created a database and connected a couple data nodes, let's go ahead
+and create a distributed hypertable. With the following commands, we'll create
+a hypertable and then distribute it.
+
+```sql
+CREATE TABLE distributed (time timestamptz, id integer, value numeric);
```

Comment:
I always get warnings when I omit the "NOT NULL" from the timestamp type.

Reply:
Cool - I think it's safe to include that, so will update